tests/test_strategies.py

```diff
@@ -198,6 +197,17 @@ def test_ridiculously_parallel_tree_search(self):
             player.tree_search(num_parallel=50)
         self.assertNoPendingVirtualLosses(player.root)
 
+    def test_long_game_tree_search(self):
```

Comment:
I think an ideal test would be one that's up against the move limit, where placing a stone in enemy territory is the "correct" move because of the end-of-game limit.

Reply:
Done!